Makefile

```diff
@@ -11,23 +11,22 @@ help:
 
 builds:
 	@echo "Building Docker Images... 🐳"
-	docker-compose build --no-cache
+	docker-compose build
 	@echo "Done! ✨"
-up:
+up: builds
 	@echo "Starting library... ⚡️ "
-	docker-compose up --build
+	docker-compose up --build -d
```

Comment:
I haven't tried running this but without the `-d` to send to the background, I would expect this line to block things from proceeding further.